core/modules/views/plugins/views_plugin_query_default.inc

```diff
@@ -289,7 +289,10 @@ class views_plugin_query_default extends views_plugin_query {
    */
   function options_submit(&$form, &$form_state) {
     $element = array('#parents' => array('query', 'options', 'query_tags'));
-    $value = explode(',', backdrop_array_get_nested_value($form_state['values'], $element['#parents']));
+    $value = backdrop_array_get_nested_value($form_state['values'], $element['#parents']);
+    if (is_string($value)) {
```

Comment:
This looks like a case, when a small comment could be helpful in the future.
"Why is this somethings a string and sometimes seemingly an array?" - That's a question that may come the next developer touching this to mind.

Reply:
@indigoxela, done. Take a look to see if this makes sense.